CollaboratedHRM/Setup/Base.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BoDi;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace CollaboratedHRM.Setup
+{
+    [Binding]
+    public class Base
+    {
+        private readonly IObjectContainer _objectContainer;
+        public IWebDriver _driver;
+        string baseUrl = "https://opensource-demo.orangehrmlive.com/";
+
+        public Base(IObjectContainer objectContainer)
+        {
+            _objectContainer = objectContainer;
+            ChromeOptions options = new ChromeOptions();
+           // options.AddExtension(@"..\..\..\extensions\AdBlock .crx");
+            _driver = new ChromeDriver(options);
+            _objectContainer.RegisterInstanceAs<IWebDriver>(_driver);
+        }
+
+        public void LoadApplicationUnderTest()
+        {
+            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(30);
+            _driver.Navigate().GoToUrl(baseUrl);
+            _driver.Manage().Window.Maximize();
+
+        }
+
```

Comment:
You need to add a method that closes the instance of your browser after test run and try not to put any comments to your code